tests/org.jboss.tools.teiid.ui.bot.test/src/org/jboss/tools/teiid/ui/bot/test/ddl/SourceForeignKey.java

```diff
@@ -0,0 +1,177 @@
+package org.jboss.tools.teiid.ui.bot.test.ddl;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.is;
+
+import org.hamcrest.core.StringContains;
+import org.jboss.reddeer.eclipse.ui.problems.ProblemsView;
+import org.jboss.reddeer.eclipse.ui.problems.ProblemsView.ProblemType;
+import org.jboss.reddeer.eclipse.ui.problems.matcher.ProblemsResourceMatcher;
+import org.jboss.reddeer.junit.requirement.inject.InjectRequirement;
+import org.jboss.reddeer.junit.runner.RedDeerSuite;
+import org.jboss.reddeer.requirements.openperspective.OpenPerspectiveRequirement.OpenPerspective;
+import org.jboss.reddeer.requirements.server.ServerReqState;
+import org.jboss.tools.teiid.reddeer.DdlHelper;
+import org.jboss.tools.teiid.reddeer.connection.ConnectionProfileConstants;
+import org.jboss.tools.teiid.reddeer.dialog.GenerateVdbArchiveDialog;
+import org.jboss.tools.teiid.reddeer.editor.RelationalModelEditor;
+import org.jboss.tools.teiid.reddeer.editor.TableEditor;
+import org.jboss.tools.teiid.reddeer.editor.VdbEditor;
+import org.jboss.tools.teiid.reddeer.perspective.TeiidPerspective;
+import org.jboss.tools.teiid.reddeer.requirement.TeiidServerRequirement;
+import org.jboss.tools.teiid.reddeer.requirement.TeiidServerRequirement.TeiidServer;
+import org.jboss.tools.teiid.reddeer.view.ModelExplorer;
+import org.jboss.tools.teiid.reddeer.wizard.imports.DDLTeiidImportWizard;
+import org.jboss.tools.teiid.reddeer.wizard.imports.ImportFromFileSystemWizard;
+import org.jboss.tools.teiid.reddeer.wizard.newWizard.VdbWizard;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ErrorCollector;
+import org.junit.runner.RunWith;
+
+@RunWith(RedDeerSuite.class)
+@OpenPerspective(TeiidPerspective.class)
+
+@TeiidServer(state = ServerReqState.RUNNING, connectionProfiles={
+		ConnectionProfileConstants.SQL_SERVER_2008_PARTS_SUPPLIER,
+})
+public class SourceForeignKey {
```

Comment:
well there are anotations @Test and also all tests have not suffix Test in this package